share/models/change.py

```diff
@@ -271,3 +242,66 @@ def _resolve_change(self):
             else:
                 change[k] = v
         return change
+
+    def _merge_objects(self, from_obj, into_obj):
+        from share.models import ShareObject
+
+        # TODO less hacky way of expressing what can be merged
```

Comment:
Oh, whoops. At first it was a set of the actual models, which couldn't be at global scope, but strings totally can.